writeup.md

```diff
@@ -0,0 +1,49 @@
+##### Cole Abbeduto - abbedutocg@msoe.edu
+##### Brad Wasilewski - wasilewski@msoe.edu
+
+### 1. Introduction
```

Comment:
Why are you doing what's in the PDF and not in the README.md like all the other labs and like how I mentioned in class?